projects/testing/src/utils/configure-test-suite.ts

```diff
@@ -0,0 +1,70 @@
+import {ComponentFixture, getTestBed, TestBed} from '@angular/core/testing';
+
+// type CallbackVoid = () => void;
+
+export const configureTestSuite = (
```

Comment:
I don't see how this hook is related to this UI Kit, and replacing the old one would be much easier if the name remains intact

Reply:
but it seems to me that then it can cause conflicts when importing from different packages
also if you need in `*.spec.ts`

```ts
import {tuiConfigureTestSuite as configureTestSuite, PageObject} from '@taiga-ui/testing';
```

Reply:
I totally understand your concern but the original repository is not in active development and has not been for years. We either migrate to this modified utility function or change the way we initialize suites. There's absolutely no reason to import the function from different packages.

Reply:
okey, no problem